config/crossSections.yaml

```diff
@@ -343,29 +343,29 @@ QCD_HT50to100:
   reference: XSDB QCD_HT50to100_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2020-09-24 04:59:47
   unc: 507400.0
 QCD_HT50to100_PSWeights:
```

Comment:
for all QCD samples: name of the XS shouldn't contain `PSWeights`, remove duplicated entries.

Reply:
ok